admin_scripts/aula_register.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    aula_register.py [URL, REGISTERCODE]
+#%
+#% DESCRIPTION
+#%    This script registers a Aula "kommegaa" screen.
+#%
+#%    It takes one optional parameter: the url for the registration to take place.
```

Comment:
This isn't actually optional, is it? The script only works if it has two parameters...

Reply:
I thought that Aula had a default URL which should have been used if a URL was not provided. That was not the case, so yes that is not optinal anymore...